pallets/domains/src/types.rs

```diff
@@ -2,14 +2,19 @@ use frame_support::pallet_prelude::*;
 use frame_support::traits::Currency;
 use sp_runtime::traits::Zero;
 
-use subsocial_support::{WhoAndWhenOf, new_who_and_when};
+use subsocial_support::{WhoAndWhenOf, new_who_and_when, Content};
 
 use super::*;
 
 pub(crate) type DomainName<T> = BoundedVec<u8, <T as Config>::MaxDomainLength>;
 pub(crate) type InnerValueOf<T> = InnerValue<<T as frame_system::pallet::Config>::AccountId>;
 pub(crate) type OuterValue<T> = BoundedVec<u8, <T as Config>::MaxOuterValueLength>;
 
+pub(crate) type RecordKey<T> = BoundedVec<u8, <T as Config>::MaxRecordKeySize>;
+pub(crate) type RecordValue<T> = BoundedVec<u8, <T as Config>::MaxRecordValueSize>;
```

Comment:
Maybe for better consistency, we need to add a prefix `Domain***` to the types? Because why is it called `DomainName` but `RecordKey`?
```suggestion
pub(crate) type DomainRecordKey<T> = BoundedVec<u8, <T as Config>::MaxRecordKeySize>;
pub(crate) type DomainRecordValue<T> = BoundedVec<u8, <T as Config>::MaxRecordValueSize>;
```